database/migrations/sql/20180604_02_create_data_types.sql

```diff
@@ -2,9 +2,22 @@
  * Enumerations
  */
 
-CREATE TYPE ENUM_gender              AS ENUM ('male', 'female', 'prefer not to say');
 CREATE TYPE ENUM_turnover_band       AS ENUM ('<£100k', '£100k-£250k', '£250k-£500k', '£500k-£750k', '£750k-£1m', '£1m-£5m', '£5m-£10m', '>£10m');
-CREATE TYPE ENUM_permission_type     AS ENUM ('own', 'own_organisation', 'own_beneficiaries');
+CREATE TYPE ENUM_permission_entity   AS ENUM (
```

Comment:
😬 https://github.com/TwinePlatform/twine-api/pull/26#discussion_r196067957

I can imagine this list changing a little once we add new tables (which is likely with the additional features we discussed with Sonja last week). IMO Leaving it as a VARCHAR for now is fine. Later on we can change it to something more structured if we really need to.